### ISSUE TITLE
TotalSegmentator 1.5.3 unable to download weights

### ISSUE THREAD

**Issue by rbumm** [wasserth/TotalSegmentator#200]
TotalSegmentator 1.5.3, which we use in the TS 3D Slicer extension per default, is currently unable to download the weights from zenodo. 

Has anything changed? 

````
Processing started
Writing input file to C:/Users/Rudolf/AppData/Local/Temp/Slicer/__SlicerTemp__2023-10-17_08+00+24.613/total-segmentator-input.nii
Creating segmentations with TotalSegmentator AI...
Total Segmentator arguments: ['-i', 'C:/Users/Rudolf/AppData/Local/Temp/Slicer/__SlicerTemp__2023-10-17_08+00+24.613/total-segmentator-input.nii', '-o', 'C:/Users/Rudolf/AppData/Local/Temp/Slicer/__SlicerTemp__2023-10-17_08+00+24.613/segmentation', '--ml', '--task', 'total']
C:\Users\Rudolf\AppData\Local\slicer.org\Slicer 5.4.0\lib\Python\Scripts\TotalSegmentator:5: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  from pkg_resources import require

If you use this tool please cite: https://doi.org/10.48550/arXiv.2208.05868

Downloading pretrained weights for Task 251 (~230MB) ...
Traceback (most recent call last):
  File "C:\Users\Rudolf\AppData\Local\slicer.org\Slicer 5.4.0\lib\Python\Scripts\TotalSegmentator", line 93, in <module>
    main()
  File "C:\Users\Rudolf\AppData\Local\slicer.org\Slicer 5.4.0\lib\Python\Scripts\TotalSegmentator", line 86, in main
    totalsegmentator(args.input, args.output, args.ml, args.nr_thr_resamp, args.nr_thr_saving,
  File "C:\Users\Rudolf\AppData\Local\slicer.org\Slicer 5.4.0\lib\Python\Lib\site-packages\totalsegmentator\python_api.py", line 155, in totalsegmentator
    download_pretrained_weights(tid)
  File "C:\Users\Rudolf\AppData\Local\slicer.org\Slicer 5.4.0\lib\Python\Lib\site-packages\totalsegmentator\libs.py", line 183, in download_pretrained_weights
    download_url_and_unpack(WEIGHTS_URL, config_dir)
  File "C:\Users\Rudolf\AppData\Local\slicer.org\Slicer 5.4.0\lib\Python\Lib\site-packages\totalsegmentator\libs.py", line 69, in download_url_and_unpack
    raise e
  File "C:\Users\Rudolf\AppData\Local\slicer.org\Slicer 5.4.0\lib\Python\Lib\site-packages\totalsegmentator\libs.py", line 56, in download_url_and_unpack
    r.raise_for_status()
  File "C:\Users\Rudolf\AppData\Local\slicer.org\Slicer 5.4.0\lib\Python\Lib\site-packages\requests\models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://zenodo.org/record/6802342/files/Task251_TotalSegmentator_part1_organs_1139subj.zip?download=1
````

**Comment by wasserth** [comment]
Zenodo did a major update of their website and it seems that they changed the way they are accepting download requests. I would have to change my download code to make it work again. Making a change to this old version is a bit difficult. The easiest for you should be to update to v1.5.6. This should work straight away since the weights in this version are no longer downloaded from zenodo.

**Comment by rbumm** [comment]
I placed a PR for the update to 1.5.6, which is working good.

**Comment by wasserth** [comment]
I just published a version 1.5.7 which now downloads the weights from github releases instead of zenodo. This should be faster and more stable.

**Comment by rbumm** [comment]
Great, it is really faster, I changed my PR to 1.5.7
